Jenkinsfile

```diff
@@ -243,6 +244,32 @@ pipeline {
           colourText("info","Deploying to BETA...")
           unstash 'zip'
           deployToCloudFoundry('cloud-foundry-bi-prod-user','bi','beta','prod-bi-ui','bi-ui.zip','manifest.yml')
+          env.APP_URL = "https://${env.DEPLOY_NAME}-${env.MODULE_NAME}.${env.CLOUD_FOUNDRY_ROUTE_SUFFIX}"
+        }
+      }
+    }
+    stage('Checking App Health') {
```

Comment:
This approach is a good as a short-term measure.  As I mentioned to @ameenhaq, we'll want to ultimately move to having system tests in a separate repo (and Jenkins job) which will get invoked after a deployment.